pyconbalkan/coc/models.py

```diff
@@ -16,5 +17,12 @@ class ResponseGuide(SingleActiveModel):
     description = MarkdownxField(null=True, blank=True)
     slug = models.CharField(unique=True, blank=True, max_length=100)
 
+    def save(self, *args, **kwargs):
+        if not self.slug:
+            self.slug = slugify(self.title)
+        elif self.slug != slugify(self.title):
```

Comment:
This will force the slug to be a slugified version of self.title, we want to be able to define it, but only assign as a default value the slugified version.

Could something like default=lambda self, x: slugify(self.title) work ?

Probably not as the field doesn't have the scope of self.

But you get my drift.